include/xtensor/xstridedview.hpp

```diff
@@ -568,6 +571,155 @@ namespace xt
     }
 #endif
 
+    class slice_vector : public std::vector<std::array<long int, 3>>
```

Comment:
You have two options:

- wraps std::vector inside slice_vector, define the interface you need, and forward the calls to the vector methods.
- inherit privately from vector, and expose the methods you need from vector via using directives:
```cpp
class slice_vector : private std::vector<std::array<long int, 3>>
{
public:

    using base_type = std::vector<std:array<long int, 3>>;
    using value_type = base_type::value_type;

    using base_type::operator[];
    using base_type::size;
    using base_type::begin;
    using base_type end;
    // ...
};
```